class0/README.md

```diff
@@ -1,16 +1,62 @@
 #Getting Ready for Olin.js
 
 ##Ubuntu
-[Startup Guide with recommended settings etc.]
+If you need help install Ubuntu check out this [guide](https://gist.github.com/benkahle/892fdb2531c5f388296a) written specifically for Olin laptops. It also includes some other useful packages and tips if you are new to Ubuntu.
 
 ##Node
-[Pull PPA instructions from https://github.com/joyent/node/wiki/installing-node.js-via-package-manager]
+We will install node from a package repository which will give us automatic updates:
+* `curl -sL https://deb.nodesource.com/setup | sudo bash -` (install curl with `sudo apt-get install curl` if needed)
+* `sudo apt-get install -y nodejs`
+* `sudo apt-get install build-essential` (needed for some npm packages)
+* To test: `node --version` should return v0.10.35 and `npm --version` should return 2.1.17
+* If your npm is not up-to-date run `sudo npm install -g npm`
```

Comment:
While software being able to update itself isn't unusual, this syntax still makes me chuckle.